### PR TITLE
Teams list

### LINKED ISSUE #39: View Teams
What has to be done:
Create a component that will be used for viewing all teams.

Why it has to be done:
So that users can search all teams and find the team that they want to view.

When the task is done:
The user can view a list of teams which they can filter on team statistics and can search for certain teams.
The team view should have pagination.

### PR DESCRIPTION
**Pull request by chf3kl9**
closes #39

### CODE REVIEW

File: src/components/teamsComponent.jsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import ApiCommunication from "./apicommunication/ApiCommunication";
+
+import MaterialTable from 'material-table';
+
+class Teams extends Component {
```
Comment (larsvandriel) [inline src/components/teamsComponent.jsx]: Maybe the class name should be changed to team list to be more specific

**Comment by lfb0801** [comment]
PR heeft geen description?

File: src/App.css [hunk 2]
```diff
@@ -40,3 +40,43 @@
   padding: 0.5rem 1rem;
   color: rgba(0,0,0,.5);
 }
+
+
+/* Moet worden toegevoegd om material icons werkend te maken. */
```
Comment (MichielLangen) [inline src/App.css]: maybe change this to English?

File: src/App.css [hunk 3]
```diff
@@ -40,3 +40,43 @@
...
+  /* Support for Safari and Chrome. */
+  text-rendering: optimizeLegibility;
+
+  /* Support for Firefox. */
+  -moz-osx-font-smoothing: grayscale;
+
+  /* Support for IE. */
+  font-feature-settings: "liga";
+}
+/* Einde: Moet worden toegevoegd om material icons werkend te maken. */
```
Comment (MichielLangen) [inline src/App.css]: maybe change to English?

File: src/views/AllTeamsView.jsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+                    }}
+                />
+            </div>
+        );
+    }
+
+}
+
+export default AllTeamsView
+
```
Comment (MindHawk) [inline src/views/AllTeamsView.jsx]: This should be in views/team

File: src/App.css (hunk 2 above)
Comment (MindHawk) [inline src/App.css]: what is this language 😭

Comment (lfb0801) [reply src/App.css]: while use multiline comment in a single line?

File: src/App.css (hunk 3 above)
Comment (MindHawk) [reply src/App.css]: wow that was my original comment do not steal

File: src/App.css (hunk 2 above)
Comment (MindHawk) [reply src/App.css]: wow that was my original comment do not steal

Comment (MichielLangen) [reply src/App.css]: *why use* not *while use*

File: src/App.css (hunk 3 above)
Comment (Papabeer04) [reply src/App.css]: Removed the comment

File: src/App.css (hunk 2 above)
Comment (Papabeer04) [reply src/App.css]: // doesn't work in css, always use /* */

**Pull request merged**